src/v/rpc/simple_protocol.cc

```diff
@@ -132,7 +132,10 @@ simple_protocol::dispatch_method_once(header h, server::resources rs) {
                   reply_buf.set_status(rpc::status::server_error);
               }
               return send_reply(ctx, std::move(reply_buf))
-                .finally([m = std::move(m)] {});
+                .finally([m, l = std::move(l)]() mutable {
+                    m->probes.latency_hist().record(
```

Comment:
why not simply move the `l` so like `m->probes.latency_hist().record(std::move(l))`

Reply:
this was just a nit.

Reply:
this is a good nit. updated